### ISSUE TITLE
Auto completion

### ISSUE THREAD

**Issue by sebacea** [ropensci/RefManageR#81]
Hi, is there an autocompletion method while writing a citation?

Thanks!

**Comment by mwmclean** [comment]
No. Can you provide more information about what you are trying to do (preferably a minimal, reproducible example) and what you would like to have happen?

**Comment by sebacea** [comment]
Thanks for the answer, in a simple Rmd template, e.g., html_document after a chunk like:

```{r setup}
options(htmltools.dir.version = FALSE)
library(RefManageR)
BibOptions(check.entries = FALSE,
           bib.style = "authoryear",
           cite.style = "authoryear",
           style = "markdown",
           hyperlink = TRUE,
           dashed = FALSE
           )
mybib <- ReadCrossRef(query="10.1007/BF00158774")
```

I would expect that while writing `r Cite(mybib, "Ba 

There is an autocompletion suggestion "ttese_1992" in order to avoid searching manually in mybib the right bibliographic key.

Maybe there is another smarter workaround that I do not know.

**Comment by drfurtado** [comment]
This would be nice.
